src/types/gecko-profile.js

```diff
@@ -130,12 +130,22 @@ export type GeckoThread = {
   stringTable: string[],
 };
 
+export type GeckoExtensionMeta = {|
+  schema: {|
+    id: 0,
+    name: 1,
+    baseURL: 2,
+  |},
+  data: Array<[string, string, string]>,
+|};
+
 export type GeckoProfile = {|
   meta: {|
     interval: Milliseconds,
     startTime: Milliseconds,
     shutdownTime: Milliseconds | null,
     abi: string,
+    extensions: GeckoExtensionMeta,
```

Comment:
If it's optional, shouldn't this be `extensions?:` then? Otherwise Flow won't catch it if we accidentally start relying on the extensions property being present. Unless I'm misunderstanding something.